pkg/agent/agent.go

```diff
@@ -221,6 +221,10 @@ func (i *Initializer) Initialize() error {
 		return err
 	}
 
+	if err := i.updateNodeMACAnnotation(); err != nil {
```

Comment:
What you think about updating Node MAC in prepareHostNetwork()?

Reply:
Works for me. Updated.